gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Wed Nov 30 23:32:35 EST 2016
+#Mon Jul 17 22:56:32 EDT 2017
```

Comment:
Did you run Gradle after that? `gradlew` and `gradle-wrapper.jar` most likely should be updated